zephyr/prj.conf

```diff
@@ -0,0 +1,58 @@
+CONFIG_EARLY_CONSOLE=y
```

Comment:
It's a bit confusing to me, why is there `prj.conf` inside a module? Any sample demonstrating supplicant should be part of `sdk-nrf`.

Reply:
File removed.